src/samples/Authorization/ManualSignIn/Program.cs

```diff
@@ -36,16 +39,28 @@
 
 var app = builder.Build();
 
+// Configure the HTTP request pipeline.
+app.UseRouting();
+app.MapPost("/api/messages", async (HttpRequest request, HttpResponse response, IAgentHttpAdapter adapter, IAgent agent, CancellationToken cancellationToken) =>
+    {
+        await adapter.ProcessAsync(request, response, agent, cancellationToken);
+    })
+    .AllowAnonymous();
```

Comment:
Allow anon should be used only in Development

Reply:
this is the eqiv to [AllowAnonymous] on the controller  its overridden by the appbuilder settings.